Maple2.Server.Game/Manager/HousingManager.cs

```diff
@@ -242,11 +246,14 @@ public class HousingManager {
         SetPlot(plotInfo);
     }
 
-    private static bool CheckCost(GameSession session, UgcMapGroup.Cost cost, bool deduct = false) {
+    private static bool CheckCost(GameSession session, UgcMapGroup.Cost cost, bool deduct = false, bool sendPacket = true) {
```

Comment:
not a bid fan of the sendPacket bool. If the packet is only for the error, can we make it return UgcMapError instead?

Reply:
see if its better like this